mule-apikit-tools/mule-apikit-scaffolder/src/test/java/org/mule/tools/apikit/ScaffolderMule4Test.java

```diff
@@ -783,6 +784,7 @@ public void testGenerateWithExchangeModules() throws Exception {
     assertEquals(1, countOccurences(s, "<apikit:console"));
   }
 
+  @Ignore
```

Comment:
Why is this being ignored? It should have an issue related

Reply:
Issue APIKIT-1724 created